topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.md

```diff
@@ -0,0 +1,721 @@
+>    - Here you have a list of all your workflows.
+>    - Your newly created workflow should be listed at the top:
+>
+>    ![`Where workflows go` list](../../images/101_foreveryone_workflow.png)
+>    ![`Your workflows` list](../../images/101_foreveryone_workflow_list.png)
+{: .hands_on}
+
+## The workflow editor
+
+We can examine the workflow in Galaxy's workflow editor. Here you can view/change the parameter settings of each step, add and remove tools, and connect an output from one tool to the input of another, all in an easy and graphical manner. You can also use this editor to build workflows from scratch.
+
+> ### {% icon hands_on %} Hands-on: Editing our workflow
+>
+> 1. Open the **workflow editor**
+>    - Click on the dropdown menu {% icon galaxy-dropdown %} (triangle icon) to the right of your workflow name.
+>
+> 2. Select **Edit** to launch the workflow editor.
+>    - You should see something like this:
+>
+>    ![Workflow editor](../../images/101_foreveryone_workflow_editor.png)
+>
+>    - When you click on a workflow step, you will get a view of all the parameter settings for that tool on the right-hand side of your screen.
+>    - You can also change the parameter settings of your workflow here, and do more advanced configuration.
+>
+> 3. **Hiding intermediate outputs**
+>    - We can tell Galaxy which outputs of a workflow are important and should be shown in our history when we run it, and which can be hidden.
+>    - By default, all outputs will be shown
+>    - Click the **asterisk** for outputs to mark them as important:
+>        - `out_file` in  **Unique** {% icon tool %}
+>        - `out_file1` in both **Group** {% icon tool%} steps
+>        - `png` in both **Scatterplot w ggplot2** {% icon tool %} steps
+>    - Now, when we run the workflow, we will only see these final outputs
+>        - i.e. the two dataset with species, the dataset with number of samples by species and the two scatterplots.
+>
+>    {% include snippets/workflow_hide_intermediate_steps.md %}
+>
+> 4. **Renaming output datasets**
+>    - When we performed the analysis manually, we often renamed output datasets to something more meaningful
+>    - We can do the same in a workflow
+>    - Let's rename all the outputs we marked with an asterisk (and thus do not hide) to more meaningful names:
+>        - `out_file` in the `Unique` tool
+>        - `out_file1` in `Group` tools,
+>        - `png` in the `Scatterplot w ggplot2` tools.
+>
+>    {% include snippets/workflow_rename_output.md %}
+>
+> 5. **Generalize workflow**
+>    - When we run this workflow on a different dataset, we may want to use some different parameters
+>    - For example for the axis labels in our plots, or the column numbers in the file
+>    - To this end, we can tell Galaxy which parameters should be set by the user every time they run the workflow
+>    - Do this for the following parameters:
+>        - **Scatterplot** {% icon tool %}: `Plot title` parameter
```

Comment:
@annefou ok, final request: could you explicitly list the parameters that should be switched to `set at runtime` here?

I was thinking, we could also change the diamond input dataset a bit to look more like the iris dataset, e.g. just the 5 columns `carat color clarity price cut`, and then column numbers can stay unchanged, and only plot titles and axis labels need to be changed for the run on the diamond data, what do you think?


`

Reply:
@shiltemann I changed what we agreed on. I think it is simpler because it is easy for users to understand why plots need to be customized. Let me know what you think and thanks a lots for your feedback.